stories/Molecules/Text/Faq/Faq.scss

```diff
@@ -4,52 +4,49 @@
 
 /* Accordian start */
 .accordion {
-  width: 100%;
+  padding: 0;
   position: relative;
-}
-
-.accordion-head {
-  border-bottom: 1px solid $color-gray-500;
-  position: relative;
-  padding: $spacing-24 0;
-  font-size: 0.87em;
-  cursor: pointer;
-  overflow: hidden;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  &::after {
-    content: "";
-    background: url(#{$img-path-Icon}/Chevron-down.svg) no-repeat center center;
-    float: right;
-    width: 20px;
-    height: 20px;
-    border-top-color: #f3f3f3;
-    transition: all 0.1s ease;
-    margin-left: $spacing-24;
-  }
-  h5 {
-    float: left;
-    margin-bottom: 0;
-  }
-  &:hover {
-    filter: alpha(opacity=80);
-    opacity: 0.8;
-  }
-  &.open {
-    &::after {
-      transform: rotateZ(180deg);
+  li {
+    border-bottom: 1px solid $color-gray-500;
+    position: relative;
+    padding: $spacing-16 0 $spacing-24;
+    cursor: pointer;
+    overflow: hidden;
+    &::before {
+      content: "";
+      background: url(#{$img-path-Icon}/Chevron-down.svg) no-repeat center center;
+      float: right;
+      width: 20px;
+      height: 20px;
+      border-top-color: #f3f3f3;
       transition: all 0.1s ease;
+      margin-left: $spacing-24;
+    }
+    h5 {
+      margin-bottom: 0;
+    }
+    &.is-active {
+      h5 {
+        margin-bottom: 10px;
+        padding-right: 24px;
+        @include devicebreak(medium) {
+          margin-bottom: 0;
+          padding-bottom: $spacing-16;
+        }
+      }
+      &::before {
+        transform: rotateZ(180deg);
+        transition: all 0.1s ease;
+      }
     }
   }
 }
 
-.accordion-body {
-  border-bottom: 1px solid $color-gray-500;
+.accordion-panel {
   padding: $spacing-32;
   height: auto;
   display: none;
-  @include devicebreak(large) {
+  @include devicebreak(medium) {
```

Comment:
@harishkumar-srijan I think we should use the small media query

Reply:
Done